doc/examples/applications/plot_solidification_tracking.py

```diff
@@ -0,0 +1,147 @@
+"""
+========================================
+Track solidification of a metallic alloy
+========================================
+
+In this example, we identify and track the solid-liquid (S-L) interface in a
+metallic sample undergoing solidification. Tracking the solidification over 
+time enables the calculatation of the solidification velocity. This is 
+important to characterize the solidified structure of the sample and will be 
+used to inform research into additive manufacturing of metals. The image 
+sequence was obtained by the Center for Advanced Non-Ferrous Structural Alloys 
+(CANFSA) using synchrotron x-radiography at the Advanced Photon Source (APS) at Argonne 
+National Laboratory (ANL). This analysis was presented at a conference [1]_.
+
+.. [1] Corvellec M. and Becker C. G. (2021, May 17-18)
+       "Quantifying solidification of metallic alloys with scikit-image"
+       [Conference presentation]. BIDS ImageXD 2021 (Image Analysis Across
+       Domains). Virtual participation.
+       https://www.youtube.com/watch?v=cB1HTgmWTd8
+"""
+
+import numpy as np
+import pandas as pd
+import plotly.io
+import plotly.express as px
+
+from skimage import filters, measure, restoration, segmentation
+from skimage.data import nickel_solidification
+
+image_sequence = nickel_solidification()
+
+y1 = 0
+y2 = 180
+x1 = 100
+x2 = 330
+
+image_sequence = image_sequence[:, y1:y2, x1:x2]
```

Comment:
```suggestion
i0, j0, i1, j1 = 0, 100, 180, 330

image_sequence = image_sequence[:, i0:i1, j0, j1]
```
Maybe the `ij` notation is better suited her and the index starting at 0...

Reply:
I would think the opposite but I'm happy to change my mind: *In general*, the `ij` notation and 0-indexing would be more standard (thinking of matrices and arrays *in general*), but I thought the `xy` notation and 1-indexing would be more physics-y, referring to spatial coordinates, ...